library/Mockery/Configuration.php

```diff
@@ -127,4 +134,27 @@ public function getInternalClassMethodParamMaps()
     {
         return $this->_internalClassParamMap;
     }
+
+    public function setObjectFormatter($class, $formatterCallback)
+    {
+        $this->_objectFormatters[$class] = $formatterCallback;
+    }
+
+    public function findObjectFormatter($class)
+    {
+        $parentClass = $class;
+        for ($classes[] = $parentClass; $parentClass = get_parent_class($parentClass); $classes[] = $parentClass);
```

Comment:
Would you mind rewriting this for loop so it is easier to understand? Thanks!

Reply:
Rewritten